ibm_catalog.json

```
@@ -0,0 +1,115 @@
+                            "key": "account_allowed_ip_addresses_enforced"
+                        },
+                        {
+                            "description": "Current MFA setting",
+                            "key": "account_iam_mfa"
+                        },
+                        {
+                            "description": "Current state of ServiceID creation restriction",
+                            "key": "account_iam_serviceid_creation"
+                        }
+                    ],
+                    "usage": ":exclamation: **Important:** Make sure that you set the `API_DATA_IS_SENSITIVE` environment variable to `true` to hide\nsensitive information before you run Terraform operations. For more information, see the generic REST API\nprovider [documentation](https://github.com/Mastercard/terraform-provider-restapi#usage):\n\nexport API_DATA_IS_SENSITIVE=true`\n\n##############################################################################\n# Config providers\n##############################################################################\n\ndata \"ibm_iam_auth_token\" \"tokendata\" {}\n\nprovider \"restapi\" {\n  uri                  = \"https:\"\n  write_returns_object = true\n  debug                = false # set to true to show detailed logs, but use carefully as it might print API key values.\n  headers              = {\n    Authorization = data.ibm_iam_auth_token.tokendata.iam_access_token\n    Content-Type  = \"application/json\"\n    if-Match      = \"*\"\n  }\n}\n\nprovider \"ibm\" {\n  ibmcloud_api_key = var.ibmcloud_api_key # pragma: allowlist secret\n}\n\n##############################################################################\n# Configure IAM Account settings\n##############################################################################\n\n# Replace \"master\" with a GIT release version to lock into a specific release\nmodule \"iam-account-settings\" {\n  source                       = \"git::https://github.com/terraform-ibm-modules/terraform-ibm-iam-account-settings?ref=main\"\n  allowed_ip_addresses         = var.allowed_ip_addresses\n  enforce_allowed_ip_addresses = true\n}"
+                }
+            ],
+            "keywords": [
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution",
+                "iam",
+                "account settings"
+            ],
+            "label": "Terraform IAM Account Settings Module",
+            "long_description": "This module configures standard IAM account settings with the recommended values, in which the default values are aligned with FSCloud requirements, and exports the values as outputs.\n\nThe module supports creating and updating settings that are applied with the terraform apply command. With objects affected by the destroy command, the module preserves the most recent setting and doesn't change objects that are configured outside of Terraform's scope.",
```

I'm not sure where the long description for the variation (flavor) appears in the catalog details.

Also, we should use the same name as what this will be categorized as in the catalog. If that's `module`, this is good, but I'm not sure.

```suggestion
            "long_description": "This module configures standard IAM account settings with recommended values. The default input values are aligned with IBM Cloud Framework for Financial Services, and exports those values as outputs.\n\nWith this module, you can create and update settings. For objects that are affected by the Terraform "destroy" command, the module preserves the most recent settings, and doesn't change objects that are configured outside of the Terraform logic.",
```

---

I just took it from the README of the module. This appears in the Partner center "Product details". I am not sure if it appears elsewhere too.